### PR TITLE
BL-19 header

### PR DESCRIPTION
**Pull request by randyhoulahan**
https://seed.bl2.staging.cbd.int/

### CODE REVIEW

**Review by rqk2k — APPROVED** [review]

File: components/icon.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<template >
+  <svg :class="{ [`chm-icon-${name}`]:true, 'chm-icon-flip': flip}" class="chm-icon" >
+    <use  v-bind="{ 'xlink:href': `#chm-icon-${name}` }"></use>
```
Comment (blaisefonseca) [inline components/icon.vue:3]: can you use `bioland-` instead of 'chm'

File: components/page/header/title-search.vue [hunk 2]
```diff
@@ -1,37 +1,254 @@
...
+    setup
+}
+
+function setup() {
+   
+    const   localePath = useLocalePath();
+    const { t  }       = useI18n();
+    const siteStore    = useSiteStore();
+
+    const { name: siteName, logo } = storeToRefs(siteStore);
```
Comment (blaisefonseca) [inline components/page/header/title-search.vue:89]: is store not already a returning ref properties?

File: components/page/header/title-search.vue [hunk 3]
```diff
@@ -1,37 +1,254 @@
...
+
+    const name = siteName.value?  siteName : ref(t('Bioland Biodiversity'))
+
+    const hasHeroImage = ref(true)
```
Comment (blaisefonseca) [inline components/page/header/title-search.vue:93]: `hasHeroImage` never gets updated, so need to make it ref?

File: i18n/en/pages/test copy.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (blaisefonseca) [inline i18n/en/pages/test copy.json:1]: delete

File: stores/site.js [hunk 5]
```diff
@@ -62,4 +71,40 @@ async function getSiteDefaultLocale(){
     const { data, error } = await useFetch(uri)
 
     return data.value
-}
\ No newline at end of file
+}
+//https://api.cbddev.xyz/api/v2023/drupal/multisite/bl2/configs/seed
+
+async function getSiteConfig(){
+    const { identifier, gaiaApi, drupalMultisiteIdentifier } = this;
```
Comment (blaisefonseca) [inline stores/site.js:78]: `API_URL` instead of gaiaApi

File: stores/site.js [hunk 6]
```diff
@@ -62,4 +71,40 @@ async function getSiteDefaultLocale(){
...
+
+    return 'https://seed.chm-cbd.net/sites/default/files/images/country/flag/xx.png'
+}
+
+async function getDefinedName () {
+    const { locale, identifier,   baseHost, defaultLocale } = this;
+    const pathPreFix = locale === defaultLocale?.locale? '' : `/${locale}`;
+    const pathLocale = pathPreFix === '/zh'? '/zh-hans' : pathPreFix;
+    const query = {jsonapi_include: 1};
+    const uri = `https://${encodeURIComponent(identifier)}${baseHost}${pathLocale}/jsonapi/site/site?api-key=636afe3fa6d502d3d7b01996b50add18`
```
Comment (blaisefonseca) [inline stores/site.js:101]: I will suggest to use nuxt server

**Pull request merged**